.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Summary
+[PLEASE REMOVE] Describe the goal of this PR and add any relevant context.
+
+[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.
+
+[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add a "Release notes: <area>" label.
```

Comment:
Please point to the CONTRIBUTING.md section that describes the "Release notes:" options, now that you've written it.